src/main/java/edu/kis/powp/jobs2d/command/SetPositionCommand.java

```diff
@@ -20,4 +20,8 @@ public void execute(Job2dDriver driver) {
 		driver.setPosition(posX, posY);
 	}
 
+	@Override
+	public void accept(Visitor visitor) {
+		visitor.visit(this);
+	}
```

Comment:
You can add a similar implementation to ICompoundCommand (default implementation)